docs/how-to/virtual_memory.rst

```diff
@@ -0,0 +1,150 @@
+.. meta::
+  :description: This chapter describes introduces Virtual Memory (VM) and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, virtual memory, virtual, memory, UM, APU
+
+.. _virtual_memory:
+
+*******************************************************************************
+HIP Virtual Memory Management
```

Comment:
Use title case for the heading (unless this is an official name for an official process)

Reply:
Fixed.